src/lib/molecules/SelectFormField.svelte

```diff
@@ -2,6 +2,7 @@
 	import Icon from '$lib/atoms/Icon.svelte'
 	import Label from '$lib/atoms/Label.svelte'
     import SelectField from '$lib/atoms/inputfields/SelectField.svelte'
```

Comment:
Waarom deze tab?

Reply:
In mijn vscode is deze tab er niet, ik denk dat er tijdens het pushen wat is gebeurd.

Reply:
GitHub kan indenting af en toe verstoren.. ignore 😁